### PR TITLE
Adds 3 new flags/banners: Port Antillia, San Colette, Mictlan (mk. II)

### PR DESCRIPTION
**Pull request by feartheblackout**
image

Was iffy on the San Colettish stars so I made them smaller but still not sure. Let me know if you have any advice for how to spruce them up.

PR remade because I am new to Smartgit.

### CODE REVIEW

**Comment by feartheblackout** [comment]
!review

File: code/game/objects/structures/flags_banners.dm [hunk 1]
```diff
@@ -1630,3 +1630,126 @@
...
+/obj/structure/sign/flag/sancolette/large/east/New()
+	..(loc, EAST)
+
+/obj/structure/sign/flag/sancolette/large/west/New()
+	..(loc, WEST)
+
+// Mictlan
+
+/obj/item/flag/mictlan
+	name = "\improper Mictlan"
```
Comment (Llywelwyn) [inline code/game/objects/structures/flags_banners.dm:1719]:
```suggestion
	name = "\improper Mictlan flag"
```

**Review by Llywelwyn — CHANGES_REQUESTED** [review]

**Comment by niennab** [comment]
These look great! Could you add a curve to the flags and banners like the others? I've made an example below with the San Colette flag. Also, I recommend using this altered Elyran star for the banner and the Fisanduh star for the flag. Stars are a nightmare in pixels.
flagfeedback1
And the zoom in:
flagfeedback2

**Comment by feartheblackout** [comment]
> These look great! Could you add a curve to the flags and banners like the others? I've made an example below with the San Colette flag. Also, I recommend using this altered Elyran star for the banner and the Fisanduh star for the flag. Stars are a nightmare in pixels. flagfeedback1 And the zoom in: flagfeedback2

Thanks for the compliment and the suggestions! Will get to them soon.

**Review by Llywelwyn — APPROVED** [review]
One thing worth saying about using the existing Elyra/Fisanduh stars is that they're 5-pointed - the San Colette star is supposed to be 7-pointed. Not sure if that's lore relevant or just because it looks cooler though, so doubt it's a dealbreaker either way.

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by niennab** [comment]
> One thing worth saying about using the existing Elyra/Fisanduh stars is that they're 5-pointed - the San Colette star is supposed to be 7-pointed. Not sure if that's lore relevant or just because it looks cooler though, so doubt it's a dealbreaker either way.

Ah gotcha. Due to the limited fidelity of sprites, sometimes it's better to have something that is emblematic of the design over trying to force the precise number of points. Alternatively a star like this this could also work in it's place instead. 
flagfeedback3

**Pull request closed without merging**